app/src/main/res/layout-sw320dp/activity_registration.xml

```diff
@@ -1,90 +1,133 @@
+            <EditText
+                android:id="@+id/edt_email"
+                android:layout_width="match_parent"
+                android:layout_height="50sp"
+                android:background="@drawable/edittext"
+                android:drawablePadding="10sp"
+                android:drawableStart="@drawable/email"
+                android:ems="16"
+                android:hint="Email ID"
+                android:inputType="textEmailAddress"
+                android:paddingEnd="10sp"
+                android:paddingStart="10sp"
+                android:textColor="@color/white"
+                android:textColorHint="@color/white" />
+
+            <EditText
+                android:id="@+id/edt_Rpassword"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
-                <EditText
-                    android:id="@+id/edt_email"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center"
-                    android:ems="16"
-                    android:hint="@string/regstr_xml_email_hint"
-                    android:inputType="textEmailAddress"
-                    android:padding="10dp" >
-                </EditText>
-            </android.support.design.widget.TextInputLayout>
-
-            <android.support.design.widget.TextInputLayout
+                android:layout_height="50sp"
+                android:layout_marginTop="15sp"
+                android:background="@drawable/edittext"
+                android:drawablePadding="10sp"
+                android:drawableStart="@drawable/password"
+                android:ems="16"
+                android:hint="Password"
+                android:inputType="textPassword"
+                android:paddingEnd="10sp"
+                android:paddingStart="10sp"
+                android:textColor="@color/white"
+                android:textColorHint="@color/white" />
+
+            <EditText
+                android:id="@+id/edt_RRepassword"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
-                <EditText
-                    android:id="@+id/edt_Rpassword"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center"
-                    android:layout_marginTop="10dp"
-                    android:ems="16"
-                    android:hint="@string/regstr_xml_pass_hint"
-                    android:inputType="textPassword"
-                    android:padding="10dp" >
-                </EditText>
-            </android.support.design.widget.TextInputLayout>
-
-            <android.support.design.widget.TextInputLayout
+                android:layout_height="50sp"
+                android:layout_marginTop="15sp"

```

Comment:
Can we please extract these hardcoded values into dimension values?

Reply:
Moving all of these to dimens.xml